apis/sdkapi/v0alpha1/metaV1.go

```diff
@@ -0,0 +1,19 @@
+package v0alpha1
```

Comment:
if this is going to hold only query schemas for the time being we can call the group `query` (`apis/query/v0alpha1/`). If the plan is to add more things here then `data` would also be fine.

Reply:
Also, why not adding this to `experimental` first?

Reply:
>  If the plan is to add more things here then data would also be fine.

The next thing is settings (app + datasource... maybe the same object?) so want to avoid "query" -- i'll use data; it is a bit awkward since that is also the package for data.Frame, but eventually Frame should be in something clients would want access to

> why not adding this to experimental first?

I have bounced between them a few times 🤣 -- the question is really how compatible we want to be with k8s.  The directory structure /apis/{group}/{version} is standard in k8s and deviating from it makes life hard.  In that world "v0*" is experimental.

Working from the assumption that we will eventually live in k8s it seems better to start using the final structure.  

But honestly I don't feel

Reply:
For the purpose of not confusing/communicating to plugin authors whether this is experimental or not it would be good to put this in the experimental package I think for now until things has settled/this is officially supported.